apps/wfo-ui/pages/processes/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { WFOProcessListPage } from '@orchestrator-ui/orchestrator-ui-components';
+import { WFOProcessesListPage } from '@orchestrator-ui/orchestrator-ui-components';
 
 export default function ProcessesPage() {
-    return <WFOProcessListPage />;
+    return <WFOProcessesListPage />;
```

Comment:
what was wrong with `WFOProcessListPage`?

I believe it was correctly named

Reply:
Yes, you're right. Done